tests/Benchmarks/System.Text.Primitives/EncodeFromUtf32toUtf16.cs

```diff
@@ -0,0 +1,76 @@
+﻿using BenchmarkDotNet.Attributes;
+using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Primitives.Tests.Encoding;
+using static System.Text.Primitives.Tests.Encoding.TextEncoderTestHelper;
+
+namespace Benchmarks.System.Text.Primitives.Benchmarks
+{
+    public class EncodeFromUtf32toUtf16
+    {
+        public IEnumerable<CodePoint> GetEncodingPerformanceTestData()
```

Comment:
Are these duplicates? Can we avoid the duplication by putting these in some helper class?